dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESContentletAPIImpl.java

```diff
@@ -1718,7 +1727,16 @@ public void delete(List<Contentlet> contentlets, User user, boolean respectFront
             if(!canLock(con, user)){
                 throw new DotContentletStateException("Content Object is locked and cannot be deleted:" + con.getIdentifier());
             }
+
+            if (con.getInode().equals(CAN_T_CHANGE_STATE_OF_CHECKED_OUT_CONTENT)) {
+                throw new DotDataException("");
+            }
         }
+
+
+          if (true){
+                System.out.println("ddd");
+          }
         List<Contentlet> perCons = perAPI.filterCollection(contentlets, PermissionAPI.PERMISSION_PUBLISH, respectFrontendRoles, user);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 134 characters long line (which is greater than 120 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00103)